runtime/runtime/src/congestion_control.rs

```diff
@@ -496,15 +472,10 @@ impl ReceiptSinkV2 {
         side_effects: bool,
         stats: &mut ChunkApplyStatsV0,
     ) -> Result<Option<BandwidthRequests>, StorageError> {
```

Comment:
Once we assume `self.bandwidth_scheduler_output` is always present, we can change the signature to return `BandwidthRequests` instead of an option.